src/CppUTest/CommandLineTestRunner.cpp

```diff
@@ -95,6 +95,7 @@ void CommandLineTestRunner::initializeTestRun()
     if (arguments_->isColor()) output_->color();
     if (arguments_->runTestsInSeperateProcess()) registry_->setRunTestsInSeperateProcess();
     if (arguments_->isRunIgnored()) registry_->setRunIgnored();
+    if (arguments_->isCrashingOnFail()) registry_->setCrashOnFail();
```

Comment:
Why do this through the registery and not go directly to UTestShell?

Reply:
I was just following the pattern for the rest of the arguments. As far as I could see, they all went through the registry so I assumed that was the design intent and did the same for the option that I added.

Reply:
Nah, I suggest to not bother with the registry. It seems it doesn't need to know about crash on failure. Adding that won't create any benefits IMHO.